hibernate-core/src/main/java/org/hibernate/persister/collection/AbstractCollectionPersister.java

```diff
@@ -744,18 +744,21 @@ protected void logStaticSQL() {
 		}
 	}
 
-	private CollectionLoader standardCollectionLoader;
+	private volatile CollectionLoader standardCollectionLoader;
```

Comment:
The `volatile` keyword is mandatory to benefit from the 'local copy' paradigm later.